modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,388 @@
+<?php declare(strict_types=1);
+/**
+ * This contains tests relevant to the API
+ *
+ * PHP Version 7
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+namespace LORIS\api\Test;
+ini_set('memory_limit', '1024M');
+
+require './vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+use Laminas\Diactoros\ServerRequest;
+use GuzzleHttp\Client;
+
+/**
+ * PHPUnit class for API testsuite
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+class ProjectsTest extends TestCase
+{
+
+    private $_client;
+    /**
+     * @var string[]
+     */
+    private $_headers;
+    private $_base_uri;
+
+    /**
+     * @var string[]
+     */
+
+    private function _login()
+    {
+        $this->_base_uri = 'https://test-loris-dev.loris.ca/api/v0.0.3/';
+        $this->_client = new \GuzzleHttp\Client(['base_uri' => $this->_base_uri]);
+        $response = $this->_client->request('POST', LORIS_URL, ['json' => ['username' => LORIS_USERNAME, 'password' => LORIS_PASSWORD]]);
```

Comment:
What is LORIS_URL?

Reply:
LORIS_USERNAME should be `UnitTester` and LORIS_PASSWORD should be `$this->validPassword`
Right @kongtiaowang ?